web/e2e-tests/drafts.test.ts

```diff
@@ -298,7 +298,7 @@ async function drafts_test(page: Page): Promise<void> {
     await create_private_message_draft(page);
     // Narrow to the conversation so that the compose box will restore it,
     // then close and try restoring it by opening the composebox again.
-    await page.click("#compose .narrow_to_compose_recipients");
+    await page.click("#compose-direct-recipient .narrow_to_compose_recipients");
```

Comment:
This more specific selector is needed since we have 2 `.narrow_to_compose_recipients` buttons now, one for dm recipients and one for channel topics, and we need to ensure we click the right one